src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -27,7 +28,6 @@
     <Compile Include="System\Threading\Tasks\TaskExtensions.CoreRT.cs" />
     <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj" />
     <TargetingPackReference Include="System.Private.CoreLib" />
-    <TargetingPackReference Include="System.Private.CoreLib.Threading" />
```

Comment:
There are multiple things broken that would need to be fixed to get this to build again for uap101aot. @SedarG has been working through them one .dll at a time - I will leave to him to deal with and re-enable.